components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/account-settings-modal.tsx

```diff
@@ -113,7 +132,25 @@ const AddAccountModal = (props: Props) => {
     onClose()
   }
 
-  const onShowPrivateKey = () => {
+  const onShowPrivateKey = async () => {
```

Comment:
non-blocker: Seems like it would be useful for us to lock the wallet here as well if the user fails 3 times

Reply:
Added this functionality here: 7cfa50a8d2583686e9fdfa234ca7444f37dcc7dd

Reply:
Looks good! thanks for adding that here